collapsiblecalendarview2/src/main/res/values/strings.xml

```diff
@@ -8,4 +8,12 @@
     <string name="friday">FRI</string>
     <string name="saturday">SAT</string>
     <string name="sunday">SUN</string>
+    <string name="content_description_ll_calendar_expand">Expand calendar</string>
+    <string name="content_description_ll_calendar_collapse">Collapse calendar</string>
+    <string name="content_description_rl_options_button">Options menu</string>
+    <string name="content_description_rl_today_button">Select today</string>
+    <string name="content_description_iv_previous_week_button">Previous week</string>
+    <string name="content_description_iv_next_week_button">Next week</string>
+    <string name="content_description_iv_previous_month_button">Previous month</string>
```

Comment:
For any content description we are using for a non-button view that we are treating like a button, we should attach "button" to the end of the content description. Even though the Accessibility Scanner app will flag it, it will still behave correctly from the user's point of view using TalkBack, which is the bigger objective here. By doing so, it will be a false positive but we'll be in a position to defend why it is flagged as such.

Reply:
In the end, we still have to satisfy name, role, value and with the current content descriptions, we are missing the role.